hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAInvocationHandler.java

```diff
@@ -110,8 +110,16 @@ private Object invokeRatis(Method method, Object[] args)
       response = ratisHandler.submitRequest(
           scmRatisRequest);
     }
-    LOG.info("Invoking method {} on target {}, cost {}us",
-        method, ratisHandler, (Time.monotonicNowNanos() - startTime) / 1000.0);
+
+    if (response.isSuccess()) {
+      LOG.trace("Invoking method {} on target {}, cost {}us",
+          method, ratisHandler,
+              (Time.monotonicNowNanos() - startTime) / 1000.0);
+    } else {
+      LOG.info("Invoking method {} on target {}, cost {}us",
+          method, ratisHandler,
+              (Time.monotonicNowNanos() - startTime) / 1000.0);
+    }
```

Comment:
```suggestion
    double elapsed = (Time.monotonicNowNanos() - startTime) / 1000.0;
    String msg = "Invoking method {} on target {}, cost {}us";
    if (response.isSuccess()) {
      LOG.trace(msg, method, ratisHandler, elapsed);
    } else {
      LOG.info(msg, method, ratisHandler, elapsed);
    }
```

Reply:
Thanks for the suggestion, this looks more clear.